cmd/thetool/common.go

```diff
@@ -49,10 +51,11 @@ type Preset struct {
 
 // File is the single file
 type File struct {
-	Name      string   `json:"name,omitempty" yaml:"name,omitempty"`
-	Countries []string `json:"countries,omitempty" yaml:"countries,omitempty"`
-	Months    *Preset  `json:"months,omitempty" yaml:"months,omitempty"`
-	Events    []Event  `json:"events,omitempty" yaml:"events,omitempty"`
+	Name      string              `json:"name,omitempty" yaml:"name,omitempty"`
+	Countries []string            `json:"countries,omitempty" yaml:"countries,omitempty"`
+	Calendars map[string][]string `json:"calendars,omitempty" yaml:"calendars,omitempty"`
+	Months    *Preset             `json:"months,omitempty" yaml:"months,omitempty"`
+	Events    []Event             `json:"events,omitempty" yaml:"events,omitempty"`
```

Comment:
Would be much better if we had two type for ```Preset``` and ```Event```

Reply:
We have, or I don't get your point.

Reply:
I meant:
```go
type Preset struct {
	Name      string   `json:"name,omitempty" yaml:"name,omitempty"`
	Countries []string `json:"countries,omitempty" yaml:"countries,omitempty"`
	Months    *Months  `json:"months,omitempty" yaml:"months,omitempty"`
}

type Events []Event 
```
however forget about it. that change needs a lot of work and probably at the end we have to marge them anyway.

Reply:
About the Months ok, you are right. but If I don't want to add a method to Events, what is the advantage of defining a new type? it decreases the readability (you need to check the ref to realize this key is an array) and adds nothing.